kuma/javascript/src/article.jsx

```diff
@@ -144,20 +144,19 @@ export default function Article({ document }: DocumentProps) {
         }
     }, [document, userData]);
 
+    const isArchive =
+        document.slug === 'Archive' || document.slug.startsWith('Archive/');
+
     return (
         /*
          * The "text-content" class and "wikiArticle" id are required
          * because our stylesheets expect them and formatting isn't quite
          * right without them.
          */
-        <div
-            id="content"
-            ref={article}
-            className="text-content"
```

Comment:
I think that taking out text-content will break stuff. I don't remember what. But I put the comment above to make sure that this wouldn't be removed.

Reply:
It's been days since this goldfish-brain made this edits but it's definitely wrong that I didn't also remove the code comment. I need to revisit this and question "Why did I remove the className?!?!"

Reply:
@schalkneethling In my latest push I took David's advice and made it so that the `text-content` className is always there.